test/tokenizer_test.rb

```diff
@@ -0,0 +1,57 @@
+require "./test/test_helper"
+require "./lib/tokenizer"
+
+class TokenizerTest < MiniTest::Test
+
+	def setup
+		@tokenizer = Tokenizer.new
+	end
+
```

Comment:
It looks like I have some invisible characters from my Atom snippet -- using \t gives inconsistent behavior.

Reply:
I fixed my snippets